corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -55,6 +55,11 @@ hqDefine("data_dictionary/js/data_dictionary", [
         self.originalResourcePropPath = fhirResourcePropPath;
         self.deprecated = ko.observable(deprecated || false);
         self.removeFHIRResourcePropertyPath = ko.observable(removeFHIRResourcePropertyPath || false);
+        self.allowedValues = ko.observableArray();
```

Comment:
Nitpick: this could be a little more concise by using `_.map` and setting `allowedValues` to the entire array instead of pushing individual values.